.changelog/3061.feature.2.md

```diff
@@ -0,0 +1,4 @@
+go/oasis-node/cmd/genesis: Allow setting staking token symbol in init
+
+Allow setting staking token's ticker symbol in `oasis-node genesis init` CLI
```

Comment:
But not the exponent?

Reply:
Good catch. I've added the ability to set the token value's exponent.